config/crd/bases/extensions.hive.openshift.io_agentclusterinstalls.yaml

```diff
@@ -142,6 +142,31 @@ spec:
                 required:
                 - name
                 type: object
+              diskEncryption:
+                description: DiskEncryption is the configuration to Enable/disable
+                  disk encryption for cluster nodes.
+                properties:
+                  enableOn:
+                    default: none
+                    description: Enable/disable disk encryption on master nodes, worker
```

Comment:
Just wonder if we have a specific scenario in mind that we want encryption enabled only for masters/workers? Is that the only granularity we plan to support?

Reply:
This is the only granularity that openshift supports, we just move strings from one place to another as a wise man once told @rollandf